plugins/linux-v4l2/v4l2-input.c

```diff
@@ -796,6 +827,13 @@ static obs_properties_t *v4l2_properties(void *vptr)
 	obs_properties_add_bool(props, "buffering",
 				obs_module_text("UseBuffering"));
 
+	obs_properties_add_bool(props, "auto_reset",
+				obs_module_text("AutoresetOnTimeout"));
```

Comment:
Include the en-US locale strings for these.

Reply:
Ohh I misunderstood the previous comment I guess. I though I should leave out all locale strings. :)

Reply:
en-US are kept updated in-repo, every other locale is handled through CrowdIn.